src/js/components/Video/Video.js

```diff
@@ -413,6 +440,20 @@ const Video = forwardRef(
   },
 );
 
+Video.defaultProps = {
+  messages: {
+    closeMenu: 'close menu',
+    fullScreen: 'full screen',
+    meter: 'video progress',
```

Comment:
Any reason we picked the name `meter`? would it make more sense to call it progress?

Reply:
No reason really, I agree that it would make more sense to call it `progressMeter`